src/devices/realsense2/realsense2Driver.h

```diff
@@ -0,0 +1,226 @@
+ * |                              |  accuracy           | double              |  Read / write   | meters         |   -           |  No                              | Accuracy of the device, as the depth measurement error at 1 meter distance             |  Note that only few realsense devices allows to set it                |
+ * |                              |  framerate          | int                 |  Read / Write   | fps            |   30          |  No                              | Framerate of the sensor                                                                |                                                                       |
+ * |                              |  enableEmitter      | bool                |  Read / Write   | -              |   true        |  No                              | Flag for enabling the IR emitter(if supported by the sensor)                           |                                                                       |
+ * |  HW_DESCRIPTION              |      -              |  group              |                 | -              |   -           |  Yes                             | Hardware description of device property.                                               |  Read only property. Setting will be disabled                         |
+ * |                              |  clipPlanes         | double, double      |  Read / write   | meters         |   -           |  No                              | Minumum and maximum distance at which an object is seen by the depth sensor            |  parameter introduced mainly for simulated sensors, it can be used to set the clip planes if Openni gives wrong values |
+ *
+ * Configuration file using .ini format, using subdevice keyword.
+ *
+ * \code{.unparsed}
+
+device       RGBDSensorWrapper
+subdevice    realsense2
+name         /depthCamera
+
+[SETTINGS]
+depthResolution (640 480)    #Note the parentesys
+rgbResolution   (640 480)
+framerate       30
+enableEmitter   true
+
+[HW_DESCRIPTION]
+clipPlanes (0.2 10.0)
+ *
+ * \endcode
+ */
+
+
+
+class yarp::dev::realsense2Driver :  public yarp::dev::DeviceDriver,
+                                     public yarp::dev::IRGBDSensor,
+                                     public yarp::dev::IFrameGrabberControls2
+{
+private:
+    typedef yarp::sig::ImageOf<yarp::sig::PixelFloat> depthImage;
+#ifndef DOXYGEN_SHOULD_SKIP_THIS
+    typedef yarp::os::Stamp                           Stamp;
+    typedef yarp::os::Property                        Property;
+    typedef yarp::sig::FlexImage                      FlexImage;
+
+#endif
+public:
+    realsense2Driver();
+    ~realsense2Driver();
+
+    // DeviceDriver
+    virtual bool open(yarp::os::Searchable& config) override;
+    virtual bool close() override;
+
+    // IRGBDSensor
+    virtual int    getRgbHeight() override;
```

Comment:
If you use `override`, usually, you don't have to specify `virtual` 😉 
That's exactly the purpose people introduced `override` attribute.

Reply:
you are right, unfortunately is plenty of these `virtual` `override`, I can change it later in a cleanup commit!